### DISCUSSION TITLE
Is there a way to generate logs?

### DISCUSSION THREAD

**Discussion by Darkforg** [MPh-py/MPh#160] [General]
Hey there and thank you for your work.

Is there a way to write a log file as the calculations are ongoing (like for example you can when running comsol through a linux command line)? The code works as intended but i don't have an indication of its progress untill it's finished.

Thanks in advance.

**Comment by john-hen** [comment]
There's no convenient way to do it, but you can have Comsol write its progress log to a file and then parse that file. See earlier discussion #55 for details.

**Comment by Darkforg** [comment]
Thank you for your quick response. Next time, I will research more thoroughly before posting.
